README.md

```diff
@@ -151,34 +148,25 @@ Needs file extension `.app`.
 Path to additional binaries that will be signed along with built-ins of Electron.
 Default to `undefined`.
 
-`entitlements` - *String*
-
-Path to entitlements file for signing the app.
-Default to built-in entitlements file, Sandbox enabled for Mac App Store platform.
-See [default.entitlements.mas.plist](https://github.com/electron-userland/electron-osx-sign/blob/master/default.entitlements.mas.plist) or [default.entitlements.darwin.plist](https://github.com/electron-userland/electron-osx-sign/blob/master/default.entitlements.darwin.plist) with respect to your platform.
-
-`entitlements-inherit` - *String*
-
-Path to child entitlements which inherit the security settings for signing frameworks and bundles of a distribution. *This option only applies when signing with entitlements.*
-See [default.entitlements.mas.inherit.plist](https://github.com/electron-userland/electron-osx-sign/blob/master/default.entitlements.mas.inherit.plist) or [default.entitlements.darwin.inherit.plist](https://github.com/electron-userland/electron-osx-sign/blob/master/default.entitlements.darwin.inherit.plist) with respect to your platform.
-
-`entitlements-loginhelper` - *String*
-
-Path to login helper entitlement file. When using App Sandbox, the inherited entitlement should not be used since this is a standalone executable. *This option only applies when signing with entitlements.*
-Default to the same entitlements file used for signing the app bundle.
-
-`entitlementsForFile` - *Function*
-
-Function that receives the path to a file and the current codesign arguments as parameters.  If you wish to override the entitlements used for this file path this function should return the absolute path to a different entitlements file.
+`optionsForFile` - *Function*
+
+Function that receives the path to a file and can return the entitlements to use for that file to override the default behavior.  The
+object this function returns can include any of the following optional keys.
```

Comment:
A table may be more descriptive and should fit some details that were in the old flag descriptions:

| Option            | Description                                                                                                                                                                                                                               | Usage Example                                                         |
|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------|
| `entitlements`    | String specifying the path to an `entitlements.plist` file. Will default to built-in entitlements files.                                                                                                                                  | `'path/to/entitlements'`                                        |
| `hardenedRuntime` | Boolean flag to enable the Hardened Runtime when signing the app. Enabled by default.                                                                                                                                                     | `false`                                                         |
| `requirements`    | String specifying the [requirements](https://developer.apple.com/library/mac/documentation/Security/Conceptual/CodeSigningGuide/RequirementLang/RequirementLang.html) that you recommend to be used to evaluate the code signature.       | `'anchor apple or anchor = "/var/db/yourcorporateanchor.cert"'` |
| `signatureFlags`  | List of [code signature flags](https://developer.apple.com/documentation/security/seccodesignatureflags?language=objc). Accepts an array of strings or a comma-separated string.                                                          | `['kSecCodeSignatureRestrict']`                                 |
| `timestamp`       | String specifying the URL of the timestamp authority server. Defaults to the server provided by Apple. Please note that this default server may not support signatures not furnished by Apple. Disable the timestamp service with `none`. | `'https://different.timeserver'`                                |